src/propIs.js

```diff
@@ -0,0 +1,28 @@
+var _curry3 = require('./internal/_curry3');
+var is = require('./is');
+var propSatisfies = require('./propSatisfies');
+
+
+/**
+ * Returns `true` if the specified object property is of the given type;
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Type
+ * @sig Type -> String -> Object -> Boolean
+ * @param {Function} type
+ * @param {String} name
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.is
+ * @see R.propSatisfies
+ * @example
+ *
+ *      R.propIs('x', Number, {x: 1, y: 2});  //=> true
+ *      R.propIs('x', Number, {x: 'foo'});    //=> false
+ *      R.propIs('x', Number, {});            //=> false
+ */
+module.exports = _curry3(function propIs(type, name, obj) {
+  return propSatisfies(is(type), name, obj);
+});
```

Comment:
It is less elegant, yes, but as you rightly pointed out we can't know what the user intended. Also,

```javascript
pipe(type => is(type), propSatisfies)
```

is closer to the ideal than

```javascript
(type, name, obj) => propSatisfies(is(type), name, obj)
```

which is our only option as things stand.